app/src/main/res/values/strings.xml

```diff
@@ -406,6 +406,8 @@
     <string name="auto_assign_labels_primary">Auto-assign as primary</string>
     <string name="label_color">Label color</string>  <!-- can be removed -->
     <string name="label_color_example">Example</string>
+    <string name="match_any_text">*ab*</string>  <!-- Probably does not need to be translated. They are symbols for a button image -->
+    <string name="match_start_of_text">Ab*</string>  <!-- Probably does not need to be translated. They are symbols for a button image -->
```

Comment:
-> untranslated_strings.xml

Reply:
though, we have also rtl languages, that should be taken care of.

Reply:
I don't think of these as strings but as an icon. I am not sure that there would be a translatable phrase for them. The important character is the *

Reply:
The issue is that on RTL languages, * should be on the other side than what we think.